prime-router/src/main/kotlin/tokens/TokenAuthentication.kt

```diff
@@ -31,7 +29,7 @@ import java.util.UUID
  *
  *
  */
-class TokenAuthentication(val jtiCache: JtiCache) : Logging {
+class TokenAuthentication : Logging {
```

Comment:
This class is mis-named.   Here "TokenAuthentication" means server2server authentication.
Both Okta and Server2server, and lots of other techniques use tokens.
Again, beyond the scope of this ticket.

Reply:
I would but I've been through so many darn names for this feature, I wanna see first if my latest name "server2server" sticks or not.

First it was "FHIR Auth", then "Token Auth" then "two legged auth", sometimes its "public-private key auth", ...